Lesson5/Task1/RationalNumber.cs

```diff
@@ -0,0 +1,194 @@
+﻿public class RationalNumber : ICloneable
```

Comment:
```suggestion
﻿public readonly struct RationalNumber : ICloneable
```
Дробь - должна быть структурой.